.github/workflows/ci.yml

```diff
@@ -76,6 +76,24 @@ jobs:
       - run: du -h index.scip
         working-directory: examples/bazel-example
 
+  bazel_aspect:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: yarn global add @bazel/bazelisk
+      - run: sbt cli/pack
+      - run: echo "$PWD/scip-java/target/pack/bin" >> $GITHUB_PATH
+      - name: Auto-index scip-java codebase
```

Comment:
IIUC, this part of the docs can be amended now? https://sourcegraph.com/github.com/sourcegraph/scip-java/-/blob/docs/getting-started.md?L332